### ISSUE TITLE
Bug: AI does not attack player with AA-Guns

### ISSUE THREAD

**Issue by Holgerhollebolle** [Project-Hatchet/public_h-60#425]
# Bug Report

## **Description:**
When you fly in a Helicopter only with Players most AA-Weapons dont attack. Only Rockets will get fired but AA Guns like ZU or tigris guns wont shoot at you. When an AI unit flys with you the other side will shoot at you.

### **Expected Behavior**
AA Guns should allways attack you when you are discovered.

### Steps to Reproduce the Issue

1. Place a MH-60M (without AI crew)
2. Place an Opfor Vehicle with a Gun like the Otokar (or Any vehicle/Turrent that normally attacks you that does not use rockets)
3. Fly to the Enemy
4. You will not get attacked.

**Comment by Holgerhollebolle** [comment]
[arma3profiling_x64_2023-10-13_14-15-30.zip](https://github.com/Project-Hatchet/H-60/files/12894092/arma3profiling_x64_2023-10-13_14-15-30.zip)
Added the Logfile

**Comment by Holgerhollebolle** [comment]
I noticed looking in the Log that this error happens when you place a heli in the editor:

```
15:41:25 Error in expression <0.5 * (HitEngine1 + HitEngine2)>
15:41:25   Error position: <HitEngine1 + HitEngine2)>
15:41:25   Error Undefined variable in expression: hitengine1
15:41:25 Error compiling '0.5 * (HitEngine1 + HitEngine2)' in 'hitengine'
15:41:25 Error in expression <(hitRWRFront + hitRWRRear)>
15:41:25   Error position: <hitRWRFront + hitRWRRear)>
15:41:25   Error Undefined variable in expression: hitrwrfront
15:41:25 Error compiling '(hitRWRFront + hitRWRRear)' in 'hitrwr'


```

**Comment by Riverman02** [comment]
Fixed with #472
